googleapiclient/http.py

```diff
@@ -24,34 +24,35 @@
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
-import StringIO
+from six import BytesIO, StringIO
+from six.moves.urllib.parse import urlparse, urlunparse, quote, unquote
```

Comment:
It's all personal preference I guess.  I don't mind using that, but I'm not sure if there is anything in the coding standard that addresses this.